.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @edouarddurieux @MattKetmo
```

Comment:
I wonder if we should use `CODEOWNERS` in public repos? (cc: @blablacar/ep)

Reply:
It's a good practice, especially in a public repository, to notify contributors that something needs attention in a low-traffic repository.

From the contributor point of view, it's cool not to dig in history for searching relevant contributors (like me when creating this pull request 😅 ).

Reply:
If we want this, it may be useful to create a new team dedicated to the repo, or at least a generic one for all external repos, e.g. @blablacar/codereviewers as some other organizations do.

Reply:
Can you create a ticket on this with what you want?

Reply:
I guess it would be more a topic/strategy to discuss with the team first–let's bring it up and hopefully decide this morning?